Formula/c/cpp-lazy.rb

```diff
@@ -0,0 +1,38 @@
+class CppLazy < Formula
+  desc "C++11 (and onwards) library for lazy evaluation"
+  homepage "https://github.com/Kaaserne/cpp-lazy"
+  url "https://github.com/Kaaserne/cpp-lazy/archive/refs/tags/v8.0.0.tar.gz"
+  sha256 "02f29ff1e6b44fc0e34cb7c651ee4b4f806c63dfc699fc135db794424fdc119f"
+  license "MIT"
+  head "https://github.com/Kaaserne/cpp-lazy.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "fmt" => :test
+
+  def install
+    args = std_cmake_args + %w[
```

Comment:
`args` is not needed to mix with `std_cmake_args`, both were used with `cmake`.

Reply:
removed